lib/src/standalone.dart

```diff
@@ -41,14 +41,20 @@ String _standaloneName;
 /// If [release] is `false`, this compiles with `--enable-asserts`.
 void _compileSnapshot({@required bool release}) {
   ensureBuild();
-
-  for (var entrypoint in entrypoints) {
-    Dart.run(entrypoint, vmArgs: [
-      if (!release) '--enable-asserts',
-      '-Dversion=$version',
-      '--snapshot=build/${p.basename(entrypoint)}.snapshot'
-    ]);
-  }
+  var existingSnapshots = <String, String>{};
+  executables.forEach((name, path) {
+    if (existingSnapshots.containsKey(path)) {
+      var existingName = existingSnapshots[path];
+      File('build/$existingName.snapshot').copySync('build/$name.snapshot');
```

Comment:
If you use Grinder's [`copyFile()`](https://pub.dev/documentation/grinder/latest/grinder.files/copyFile.html), this will automatically be logged.

Reply:
Grinder's `copyFile` only works for copying a file with the same name into a new directory, so I'll log it manually